### PR TITLE
Upgrading Jquery to 3.4.1, jquery-ui to 1.12.1 and bootstrap to 3.4.1 in jaggary apps

### PR DESCRIPTION
**Pull request by piraveena**

### CODE REVIEW

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-dashboard-designer.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var resources = function (page, meta) {
     return {
         template: 'portal-dashboard-designer.hbs',
-        js: ['portal-gadgets-js.jag','shindig.js', 'UESContainer.js', 'jquery-ui-1.10.1.custom.min.js', 'jquery.gridster.with-extras.min.js', 'portal-dashboard-designer.js', 'alert.js'],
+        js: ['portal-gadgets-js.jag','shindig.js', 'UESContainer.js', 'jquery-ui-1.12.1.custom.min.js', 'jquery.gridster.with-extras.min.js', 'portal-dashboard-designer.js', 'alert.js'],
```
Comment (jeradrutnam) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-dashboard-designer.js]: Fix character length

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-editor.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var resources = function (page, meta) {
     return {
         template: 'portal-editor.hbs',
-        js: ['jquery-ui-1.10.1.custom.min.js', 'jquery.nicescroll.min.js', 'elfinder.full.js','portal-gadgets-js.jag','shindig.js', 'UESContainer.js',
+        js: ['jquery-ui-1.12.1.custom.min.js', 'jquery.nicescroll.min.js', 'elfinder.full.js','portal-gadgets-js.jag','shindig.js', 'UESContainer.js',
```
Comment (jeradrutnam) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-editor.js]: Fix character length

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-dashboard-browser.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 var resources = function (page, meta) {
     return {
-        js: ['portal-dashboard-browser.js', 'jquery-ui-1.10.1.custom.min.js', 'elfinder.full.js', 'jquery.nicescroll.min.js' , 'jquery.tokeninput.js', 'jquery.validate.js'],
+        js: ['portal-dashboard-browser.js', 'jquery-ui-1.12.1.custom.min.js', 'elfinder.full.js', 'jquery.nicescroll.min.js' , 'jquery.tokeninput.js', 'jquery.validate.js'],
```
Comment (jeradrutnam) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/themes/portal/helpers/portal-dashboard-browser.js]: Fix character length

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/gadget.xml [hunk 4]
```diff
@@ -116,16 +116,15 @@
...
 	        $("#message").append(element);
             message({content:'No authorized applications found !', type:'info', cbk:function () {
             cancel();
             } });
             
 }
 else {
-
+    json = $.parseJSON(data);
                                     drawPage();
```
Comment (jeradrutnam) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/gadget.xml]: Fix indentation

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/gadget.xml [hunk 5]
```diff
@@ -116,16 +116,15 @@
 
 
 
-      json = $.parseJSON(data);
-      if(json == null){
+      if(data == ""){
 	        $("#message").append(element);
```
Comment (jeradrutnam) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/gadget.xml]: Fix indentation

**Comment by piraveena** [comment]
Fixed comments with https://github.com/wso2/product-is/pull/6773

**Pull request merged**